tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -540,6 +540,27 @@ def test_log_softmax_invalid(device, a_shape, axis, float_dtype):
         return chainerx.log_softmax(a, axis=axis)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+class testSigmoid(op_utils.NumpyOpTest):
```

Comment:
Would it be possible to add some parameterizations similar to the [tanh test](https://github.com/chainer/chainer/blob/master/tests/chainerx_tests/unit_tests/routines_tests/test_math.py#L559-L563) especially since C++ tests most likely will be removed once Python tests have enough coverage.